lib/plurimath/cli.rb

```diff
@@ -74,11 +71,72 @@ def convert
       puts output_text
     end
 
+    desc "render", "Generate png, svg or pdf file of the input"
+
+    option :input,
+           aliases: "-i",
+           desc: "Input value should be in quoted string"
+
+    option :input_format,
+           aliases: "-f",
+           default: "asciimath",
+           desc: "Input format should be in quoted string => \"asciimath\""
+
+    option :file_path,
+           aliases: "-p",
+           desc: "Reads input from a file instead of the command line input. Use this for larger inputs or when input contains special characters.",
+           force: :boolean
+
+    option :output,
+           aliases: "-o",
+           required: true,
+           desc: "Expected file name with extension for output. Extension must be png, svg or pdf only."
+
+    option :display_style,
+           aliases: "-d",
+           desc: "DisplayStyle is only supported for MathML conversion, Boolean only",
+           force: :boolean
+
+    def render
+      warn_and_exit("missing generator argument --input or --file-input") unless input_string
+
+      output = options[:output]
+      warn_and_exit("wrong generator argument output value") unless %w[png svg pdf].include?(output.split(".").last)
```

Comment:
I think we should use [`File.extname(output)`](https://ruby-doc.org/core-2.6.3/File.html#method-c-extname) method for getting extension name.